templates/saves.ejs

```diff
@@ -29,13 +45,35 @@
             > 
                 Load
             </button>
-            <% console.log(locals) %>
-            <% if (saveData[`save${i}`]) { %> <%= saveData[`save${i}`].date %> <% } %>
+            <!-- <% console.log(locals) %> -->
+            <input class="save-name-input" style="flex-grow: 1;" type="text" 
+            id="save_name_<%= i %>"
+            value="<%= saveData[`save${i}`].name %>"
+            onblur="renameLocalStorageSave('save<%= i %>', 'save_name_<%= i %>')"
```

Comment:
@thuiop 
I was thinking of adding the following attributes to this input element (that updates the name of the save). Do you think this would cause any side effects? I didn't see any so far when I experimented with it.

`hx-trigger='blur'
hx-post
hx-ext="submitlocalstorage"
hx-target="#content"
hx-swap="innerHTML"`

Reply:
I want to do so to have some form of double binding between the input value and the localstorage save's name

Reply:
Seems like a good idea. You probably should then do 
```suggestion
            hx-on::beforerequest="renameLocalStorageSave('save<%= i %>', 'save_name_<%= i %>')"
```
to be sure it runs before the request is sent.